### DISCUSSION TITLE
After moving over my pocketbase project from mac to linux my admin signing to admin ui is not working.

### DISCUSSION THREAD

**Discussion by MattHamp21** [pocketbase/pocketbase#2150] [Q&A]
I moved one of my projects over to a Linux Vm using apache. When I sign into my pocketbase admin Ui on my mac it works, but when I try to sign in to my admin UI on my VM the same email and password don't work. I attempted to get my password by pressing forgot password but it does not send a email.  Is there any way to see my admin email and password from my code? Or is there possibly a base pocketbase sign in and password. Let me know if you need more information. I would love some help understanding this because I'm new to programming and this is my first project. Thanks!

**Comment by ganigeorgiev** [comment]
Did you move the `pb_data` directory from the mac to the linux vm too? If you want to have the same data from the mac on the linux vm, you'll have to copy it too, otherwise PocketBase will initialize with a blank state.

If you forgot your admin password you can run the following command to temporary reset it to `123456`:

```sql
sqlite3 pb_data/data.db 'UPDATE _admins SET passwordHash="$2a$10$xWbkSks7OiTmrri2DU6QSe6OAekcdqkOrkcI1wi4uiv9SU2HTlHoq" where email="test@example.com"'
```

_(if you don't remember your email, you can remove the `where` part from the query to reset the password for all admins)._
